README.md

```diff
@@ -58,6 +58,18 @@ However, `shx` is ideal for one-liners inside `package.json`:
 }
 ```
 
+## Contributing
+
+### Releasing
```

Comment:
I think this should be in `CONTRIBUTING.md`, it's not useful to most README users.